crates/runtime/src/instance.rs

```diff
@@ -265,7 +132,7 @@ cfg_if::cfg_if! {
             where
                 H: 'static + Fn(winapi::um::winnt::EXCEPTION_POINTERS) -> bool,
             {
-                self.instance_mut().signal_handler = Box::new(handler) as Box<SignalHandler>;
+                drop(self.instance().signal_handler.replace(Box::new(handler)));
```

Comment:
Here and in a few other places, is `.replace` where the result immediately dropped different than `.set`?

Reply:
Heh it is not, this is "Alex still hasn't forgotten the days where `set` required `T: Copy`". I'll update this to all be `set` which avoids the confusing `drop` as well.